packages/vitest/src/node/workspace/resolveWorkspace.ts

```diff
@@ -67,6 +68,12 @@ export async function resolveWorkspace(
       : options.extends === true
         ? (vitest.vite.config.configFile || false)
         : false
+
+    // Merge --browser.<option> with projects' test.browser options
+    const browser = (cliOptions.browser || options.test?.browser)
+      ? { ...options.test?.browser, ...cliOptions.browser }
+      : undefined
```

Comment:
this will break if config has the browser field

```ts
/// config file
export default { test: { browser } }

/// cli
createVite({
  test: {
    browser: undefined
  }
})
```

it should pass down `{}` instead

should there be a test for this?